### ISSUE TITLE
[Feature Request] Add "unite extension" options

### ISSUE THREAD

**Issue by sketet** [home-sweet-gnome/dash-to-panel#631]
Is it possible to include in dash to panel some feature of the extension "unite" (https://extensions.gnome.org/extension/1287/unite/) or similar?

- Customize spacing of the icon in the panel

- Remove dropdown arrows

- Remove window title bar

- Add window control buttons

And finally, move the appmenu close to the taskbar.

**Comment by charlesg99** [comment]
You could use other extensions that are know to work with dash to panel for most of these.

> Customize spacing of the icon in the panel

Unless I don't understand what you mean, I don't think the spacing could be more configurable than what it is now. Did you try playing with the "App icon margin" from the "Style" tab in the settings?


> Remove window title bar 
Add window control buttons

See the [No Title Bar](https://extensions.gnome.org/extension/1267/no-title-bar/) extension.


> move the appmenu close to the taskbar.

Not sure what you mean.

Thanks!

**Comment by sketet** [comment]
Yes, I use other extensions. 
It would be great include these extensions in order to manage all the possible settings about the panel in one extension and maybe don't meet redudant or incompatible options inside different extensions.

- Customize spacing of the icon in the panel
[Unite](https://extensions.gnome.org/extension/1287/unite/) or [Status Area Horizontal Spacing](https://extensions.gnome.org/extension/355/status-area-horizontal-spacing/)

- Remove window title bar & Add window control buttons
Like you suggest [No Title Bar](https://extensions.gnome.org/extension/1267/no-title-bar/), but for the control buttons I prefer to use "Unite" because I can choose the theme, unfortunately don't work anymore.

- Move the appmenu close to the taskbar
That's what I mean:
![Schermata del 2019-04-27 14-34-51](https://user-images.githubusercontent.com/34894217/56849882-cc499d80-68fa-11e9-8bd0-91aad4c04c06.jpg)
It's not very important, but could be a further option.

**Comment by charlesg99** [comment]
For the icons spacing, have a look at the "Style" and "Fine-Tune" tabs of dash to panel settings. For the rest, I would reiterate what I proposed in #625 and suggest that you open a Unite issue, so it accommodates dash to panel's allocation. 

A hackish/poor solution to have both extensions (dash to panel and Unite) coexist would be to make sure that the folder `~/.local/share/gnome-shell/extensions/dash-to-panel@jderose9.github.com` is physically created **after** the `~/.local/share/gnome-shell/extensions/unite@hardpixel.eu`. Best way to do this is to delete both folders, install Unite then install dash to panel and finally restart gnome-shell. This way, dash to panel will be loaded after Unite and the allocation should work as expected. I just tested and it works fine, although only on the primary monitor as Unite solely affects the default gnome-shell top panel. @onhao, maybe this would help you as well.

Thanks!
